test/functional/mempool_expiry.py

```diff
@@ -26,44 +26,48 @@
 class MempoolExpiryTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
+        self.setup_clean_chain = True
 
     def test_transaction_expiry(self, timeout):
         """Tests that a transaction expires after the expiry timeout and its
         children are removed as well."""
         node = self.nodes[0]
+        self.wallet = MiniWallet(node)
+
+        # Ensure the mempool starts empty.
+        node.generate(1)
+        assert_equal(set(node.getrawmempool()), set())
+        # Add enough mature utxos to the wallet so that all txs spend confirmed coins.
```

Comment:
this comment is not true. The "trigger" txs are chained txs. More inputs are needed.

Reply:
That's a good point, addressed in most recent commit